hadoop-ozone/dev-support/intellij/ozone-site.xml

```diff
@@ -71,4 +71,24 @@
     <name>datanode.replication.port</name>
     <value>0</value>
   </property>
+  <property>
+    <name>ozone.scm.ratis.enable</name>
+    <value>false</value>
+  </property>
```

Comment:
Seems local changes is pushed, plz revert

Reply:
No this is for dev support , so these changes will be helpful. Not done by mistake @sumitagrawl